packages/create-sitecore-jss/src/templates/angular-xmcloud/src/app/components/promo/promo.component.html

```diff
@@ -0,0 +1,49 @@
+<ng-container [ngTemplateOutlet]="variant"></ng-container>
+
+<ng-template #default>
+  <div class="component promo {{ styles }}" [attr.id]="id">
+    <div class="component-content">
+      <div class="field-promoicon">
+        <img *scImage="rendering.fields.PromoIcon" />
+      </div>
+      <div class="promo-text">
+        <div>
+          <div class="field-promotext">
+            <div *scRichText="rendering.fields.PromoText"></div>
+          </div>
+        </div>
+        <div class="field-promolink">
+          <a *scLink="rendering.fields.PromoLink"></a>
```

Comment:
scLink currently renders empty `<a></a>` tag when link field's href and text are empty. Let's include the fix for that in the PR for parity with nextjs.

Reply:
merged your contribution